README.md

```diff
@@ -119,7 +119,8 @@ The following options are supported across all subcommands:
     The password for the sql-user.  (default: autoconfigured from /etc/opscode/chef-server-running.json)
 
   * `--purge`:
-    Whether to sync deletions from backup source to restore destination. (default: false)
+    Whether to sync user deletions from backup source to restore destination. (default: false)  
+    Note that this can only be used to sync **_user_** deletions.
```

Comment:
I think we found that this isn't true.  Because chef-fs has built in behaviors if we pass in the `:purge` option from the CLI (which we do).

Reply:
@marcparadise Oh?  So we pass in that option... but it's broken?

Reply:
No - it is not broken, it gets sent directly to chef-fs https://github.com/chef/chef/blob/main/lib/chef/chef_fs/file_system.rb#L267 Currently it is not working as expected for acls.

Reply:
Since we will be purging  the node and client ACLs also, this docs update may not be necessary.  Will move it back to 'in progress' and label it as WIP until we clarify what to do.